### PR TITLE
Add an admin interface for adding new tags to existing badges.

### PR DESCRIPTION
**Pull request by ralphbean**

### CODE REVIEW

File: tahrir/templates/badge-base.mak [hunk 1]
```diff
@@ -44,6 +44,28 @@
...
+                required
+                style="width: 100%;" />
+            </div>
+            <div class="grid-50">
+              <input class="pretty-submit"
+                style="height: 30px; width:100%"
+                name="add"
+                type="submit"
+                value="Add" />
+            </div>
```
Comment (pypingou) [inline tahrir/templates/badge-base.mak]: No csrf protection?

**Comment by pypingou** [comment]
One question, otherwise :+1: 

Btw, can tag be removed?

**Comment by ralphbean** [comment]
> Btw, can tag be removed?

Nope.  :)  You'd have to edit the db by hand.  (Before this PR, you'd have to edit the db by hand in order to add a tag also, so.. progress.)

**Comment by pypingou** [comment]
On Thu, Apr 02, 2015 at 10:18:28AM -0700, Ralph Bean wrote:
>      Btw, can tag be removed?
> 
>    Nope. :) You'd have to edit the db by hand. (Before this PR, you'd have to
>    edit the db by hand in order to add a tag also, so.. progress.)

One approach I took on pagure is that:
the field contains all the current value:
- add one, it gets added
- remove one, it gets removed

Logic is here:
http://pagure.dev.fedoraproject.org/pagure/blob/master/pagure/lib/__init__.py#_1353

**Pull request merged**